.gitmodules

```diff
@@ -0,0 +1,15 @@
+[submodule "tools/dockerfile-llb-frontend/test/apps/app-httpreply"]
+	path = tools/dockerfile-llb-frontend/test/apps/app-httpreply
+	url = git@github.com:unikraft/app-httpreply.git
+[submodule "tools/dockerfile-llb-frontend/test/apps/app-redis"]
+	path = tools/dockerfile-llb-frontend/test/apps/app-redis
+	url = git@github.com:unikraft/app-redis.git
+[submodule "tools/dockerfile-llb-frontend/test/apps/app-nginx"]
+	path = tools/dockerfile-llb-frontend/test/apps/app-nginx
+	url = git@github.com:unikraft/app-nginx.git
+[submodule "tools/dockerfile-llb-frontend/test/apps/app-sqlite"]
+	path = tools/dockerfile-llb-frontend/test/apps/app-sqlite
+	url = git@github.com:unikraft/app-sqlite.git
+[submodule "tools/dockerfile-llb-frontend/test/apps/app-helloworld"]
+	path = tools/dockerfile-llb-frontend/test/apps/app-helloworld
+	url = git@github.com:unikraft/app-helloworld.git
```

Comment:
I don't think we should add git submodules to the repository, i think it over complicates the repo. Since we also plan on migrating also to a catalog approach these repositories will soon be deprecated.

Instead, we can use kraft and git to simply gather these repos.

Reply:
Done